sql/src/main/java/io/crate/analyze/repositories/RepositorySettingsModule.java

```diff
@@ -60,7 +60,7 @@
         Collections.<String, SettingsApplier>emptyMap(),
         ImmutableMap.<String, SettingsApplier>builder()
             .put("uri", new SettingsAppliers.StringSettingsApplier(new StringSetting("uri")))
-            .put("user", new SettingsAppliers.StringSettingsApplier(new StringSetting("user")))
+            .put("security.principal", new SettingsAppliers.StringSettingsApplier(new StringSetting("security.principal")))
             .put("path", new SettingsAppliers.StringSettingsApplier(new StringSetting("path")))
             .put("load_defaults", new SettingsAppliers.BooleanSettingsApplier(new BoolSetting("load_defaults", true)))
             .put("conf_location", new SettingsAppliers.StringSettingsApplier(new StringSetting("conf_location")))
```

Comment:
this should be removed, right?

Reply:
this was the fix. renaming "user" to "security.principal"

Reply:
yes got that. but this settings is not supported anymore as also the new changes entry tells. so the setting must be removed, otherwise it will be still supported by the settings system.

Reply:
ah gotcha, good catch (I thought you were commenting on the "user" rename). Borked this conflict resolution :) . Pushed the fix now